docs/tutorials.rst

```diff
@@ -194,26 +200,28 @@ example.
 
    import random
    import string
-
+   
```

Comment:
Adding the blank lines is probably fine, but better to avoid trailing whitespace, which will cause flux when this file is resaved by an author who trims trailing whitespace on save.